packages/adapters-library/src/scripts/buildMetadata.ts

```diff
@@ -93,6 +81,50 @@ export function buildMetadata(
                 }
               | undefined
 
+            if (adapter.adapterSettings.version === 2) {
```

Comment:
open to suggestions here...ideally we want to check if .getProtocolTokens has cache decorator applied...or maybe we dont need to check and we just call getProtocolTokens(true)

Reply:
Passing extra parameters won't make it fail if it's not used, so it's fine to do it this way.

We cannot directly tell if `getProtocolTokens` is decorated, so I think this is good enough.